### PR TITLE
DOCSP-22726 c2c install

### PR DESCRIPTION
**Pull request by davemungo**
This PR is based on the work Kenneth did in an earlier [PR](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/8/files)

JIRA
[DOCSP-22726](https://jira.mongodb.org/browse/DOCSP-22726)

STAGING:
[Top level](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install-v2/installation/)
[Linux](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install-v2/installation/install-on-linux/)
[macOS](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install-v2/installation/install-on-macOS/)

### CODE REVIEW

File: source/installation.txt [hunk 1]
```diff
@@ -4,10 +4,23 @@
...
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+These documents provide instructions to install {+c2c-full-product-name+}.
+
+:ref:`c2c-install-linux`
+   Install the official builds of {+c2c-product-name+} on Linux-based operating systems.
+
+:ref:`c2c-install-macos`
```
Comment (jmd-mongo) [inline source/installation.txt:18]: [broken ref]

Looks like this ref is not resolving properly:

Screen Shot 2022-05-25 at 2 09 01 PM
.

File: source/includes/step-update-path.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  variable, such as ``/usr/local/bin``. (Update
+  ``/path/to/mongosync/dist`` to reflect the location where you
+  extracted the ``tar`` file.)
+
+  .. code-block:: bash
+
+     sudo cp /path/to/mongosync/dist/mongosync /usr/local/bin/
+
+- Create symbolic links to the ``mongosync`` binary from a
+  directory such as``/usr/local/bin`` that is already in your
```
Comment (jmd-mongo) [inline source/includes/step-update-path.rst:15]: [spacing]

There ought to be a space between "as" and "``/usr/local/bin``".

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the changes, @davemungo. I've noted two issues, but this looks solid from a copy and IA standpoint!

Thanks,
Joe

**Review by jmd-mongo — APPROVED** [review]
This one LGTM at this point. Thanks!

**Review by alan-zheng-mdb — CHANGES_REQUESTED** [review]

**Review by davemungo — COMMENTED** [review]
@alan-zheng-mdb thanks for the feedback, this is ready for another pass.

**Review by alan-zheng-mdb — APPROVED** [review]
lgtm

**Pull request merged**